src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -120,9 +120,7 @@ static inline int MPIDI_OFI_win_init(MPI_Aint length,
                                      int disp_unit,
                                      MPIR_Win ** win_ptr,
                                      MPIR_Info * info,
-                                     MPIR_Comm * comm_ptr,
-                                     int create_flavor,
-                                     int model)
+                                     MPIR_Comm * comm_ptr, int create_flavor, int model)
```

Comment:
Given that all of the other parameters are on their own lines, that should probably continue here.